### PR TITLE
Addition my profile to LinkFree

### PR DESCRIPTION
**Pull request by anirudhp26**

### CODE REVIEW

**Review by imlakshay08 — APPROVED** [review]
Congratulation on your first PR! Looks great. Keep it up mate!

A basic suggestion - try to add description if you can, as template is already provided for LinkFree PRs'. You just need to check the boxes provided.

**Comment by anirudhp26** [comment]
@imlakshay08 Thanks Mate ! , I am looking forward to contribute to the latest feature requested by Sir Eddie ( Event Page ) 
would you like to help ?

**Comment by imlakshay08** [comment]
> @imlakshay08 Thanks Mate ! , I am looking forward to contribute to the latest feature requested by Sir Eddie ( Event Page ) would you like to help ?

Sure buddy! thank you for asking.

**Comment by anirudhp26** [comment]
@imlakshay08 Cool then, I am just waiting for getting assigned to the Feature

**Review by EmmaDawsonDev — APPROVED** [review]

File: public/data/anirudhp26.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (Panquesito7) [inline public/data/anirudhp26.json:1]: There seems to be an extra empty line here.
```suggestion
```

I'm not sure this is gonna be fixed by the CI automatically.

**Comment by anirudhp26** [comment]
@Panquesito7 Thanks mate!, It got updated

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉

**Pull request merged**